airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/ad_group_ads.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
+    "customer.id": {
```

Comment:
I don't find any documentation that states the uniqueness of IDs between different accounts, so customer ID should be added to all streams where it can be requested. For other resources, I will add parent resource id to the stream schema.

Reply:
Will be fixed in different pr: https://github.com/airbytehq/airbyte/pull/30091